src/components/Facility/ConsultationDoctorNotes/index.tsx

```diff
@@ -79,9 +79,13 @@ const ConsultationDoctorNotes = (props: ConsultationDoctorNotesProps) => {
       },
     });
 
-    if (res?.status === 201) {
+    if (res?.status === 201 && data) {
       Notification.Success({ msg: "Note added successfully" });
       setNoteField("");
+      setState((prevState) => ({
+        ...prevState,
+        notes: [data, ...prevState.notes],
+      }));
```

Comment:
this is not the right way. This solves only the problem for the user in the same tab. 

However another user (User A) who has this discussion notes opened won't be receiving the message from (User B)

Reply:
<img width="1390" alt="image" src="https://github.com/user-attachments/assets/5d22f4af-c79a-483f-ba3f-2ac436ce8272">

Reply:
@rithviknishad this seems like a backend issue, checked push messaging the application tab in the inspector, did not receive any incoming notifications

cc: @JavidSumra